tools/nvm/nvm1.md

```diff
@@ -0,0 +1,27 @@
+# Definition
+
+### What is **nvm**?
+**nvm** is a **node version manager** and is used to upgrade node version [nvm GitHub](https://github.com/creationix/nvm). It is a simple bash script to manage multiple active **node.js** versions.

```

Comment:
I cannot understand exactly the rest of the 1st sentence, please rephrase ! :slightly_smiling_face: